front/src/utils/date.ts

```diff
@@ -162,3 +164,9 @@ export function convertIsoUtcToLocalTime(isoUtcString: string): string {
 export function addDurationToIsoDate(startTime: string, duration: string) {
   return dayjs(startTime).add(ISO8601Duration2sec(duration), 'second').format();
 }
+
+export function transformDateWithoutYear(dateTime: string) {
```

Comment:
> We should have unit tests to avoid regression.

Is there any reason not to test it?
In our case, the affected function is particularly easy to test.